gateway-ha/src/main/resources/gateway-ha-persistence.sql

```diff
@@ -18,7 +18,7 @@ CREATE INDEX query_history_created_idx ON query_history(created);
 
 CREATE TABLE IF NOT EXISTS resource_groups (
     resource_group_id BIGINT NOT NULL AUTO_INCREMENT,
-    name VARCHAR(250) NOT NULL UNIQUE,
+    name VARCHAR(250) NOT NULL,
```

Comment:
Why are we changing the MySQL sql for this PR?

Reply:
I updated it to match the schema in trino for consistency - https://github.com/trinodb/trino/blob/master/plugin/trino-resource-group-managers/src/main/resources/db/migration/mysql/V2__add_resource_groups.sql

Happy to revert if unique names need to be enforced for gateway.

Reply:
One use case for non-unique names I have had in the past with Trino is storing the same resource group definitions for different environments in the same database. This is possible since resource groups are determined by the `environment` column of the `resource_groups` table. So you could use a single database and have the same resource group definitions for multiple environments.

Reply:
I'm not sure. Trino clusters require an environment name set, while the gateway could front backends with different environment names. So environment could be less meaningful in the gateway context. OTOH, this is defined by the user, and they can choose not to complicate things. 

I guess being able to import a set of policies from Trino to the gateway probably wins out.